test-cases/javascript/interactive/multi-step-request-event-attribute.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<body>
+    <title>CrawlMaze - Testbed for Web Crawlers</title>
+    <h1> multi-step GET method </h1>
+    <input type="button" onClick="showSecretButton();" value='show secret button'/>
+    <input type="button" onClick="doNothing();" value='decoy button'/>
+    <p>
+        When the 'show secret button' button is pressed, a new button is created in JavaScript. This new button executes a function onClick, which executes a GET request via JavaScript fetch.
+    </p>
+</body>
+
+<script> 
+    function showSecretButton() {
+      if (document. getElementById('secretbutton') == null){
+          var tag = document.createElement("button");
+          tag.textContent = "click this secret button";
+          tag.id = 'secretbutton';
```

Comment:
Maybe put the `'secretbutton'` into a constant?

Reply:
I don't think I understand what what exactly what you mean. Do you think you could elaborate? In this code, I check if the secretbutton has already been added by checking for its id. If it has not been created, I generate the button and set its id. In theory, I could potentially define a variable that changes based on whether the button has been generated or not (instead of checking by id), but I'm not sure I understand where constants might come in.

Reply:
Ah I see, that makes sense, thanks! I just changed it.